contracts/L1/gateway/L1Migrator.sol

```diff
@@ -510,14 +510,11 @@ contract L1Migrator is
         bytes32 _structHash,
         bytes memory _sig
     ) private view {
-        require(
-            _l2Addr != address(0),
-            "L1Migrator#requireValidMigration: INVALID_L2_ADDR"
-        );
+        require(_l2Addr != address(0), "L1Migrator#validMigr:INV_L2_ADR");
```

Comment:
Instead of abbreviating in this way which results in sometimes abbreviating the prefix before the colon and sometimes abbreviating the part after the colon what about just dropping the prefix i.e. `contract#function` altogether? The convenience slightly decreases in the scenario where there is a nested function call and there are revert reasons with the same language, but you can still do some debugging to identify the triggering require statement. And this way you just need to abbreviate the main message if needed. So, in this case just `INVALID_L2_ADDR`.